iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ReplaceTeePointByVoltageLevelOnLine.java

```diff
@@ -199,7 +206,7 @@ public void apply(Network network, boolean throwException,
             newLine2Adder.setBus1(bus2.getId());
             newLine2Adder.setConnectableBus1(bus2.getId());
         } else if (topologyKind == TopologyKind.NODE_BREAKER) {
-            BusbarSection bbs = network.getBusbarSection(bbsOrBusId);
+            BusbarSection bbs = getBusBarSectionFromVoltageLevel(tappedVoltageLevel, bbsOrBusId);
```

Comment:
Acutally, here you can do `tappedVoltageLevel.getNodeBreakerView().getBusbarSection(bbsOrBusId)`

Reply:
it does not seem to work as expected as stated in the comment section

"tappedVoltageLevel.getNodeBreakerView().getBusbarSection(bbsOrBusId)" will return the busBarSection from the network even if it does not belong to "tappedVoltageLevel"